.github/workflows/mutation.yml

```
@@ -1,31 +1,25 @@
+      - "docs/**"
+      - "README.md"
 
-  push:
-    paths-ignore:
-      - 'docs/**'
-      - 'README.md'
-      - 'CHANGELOG.md'
-      - '.gitignore'
-      - '.gitattributes'
-      - 'psalm.xml'
+  push: *ignore-paths
```

actionlint (1.7.9)

"push" section is alias node but mapping node is expected

(syntax-check)